indico/modules/events/templates/management/_event_location.html

```diff
@@ -0,0 +1,22 @@
+{% macro render_event_location_info(data) -%}
+    <div id="event-location">
+        <dl class="i-data-list">
+            <dt>{% trans %}Venue{% endtrans %}</dt>
+            <dd>
+                {% if data.venue -%}
```

Comment:
I don't think this is correct. This won't show anything in case of a manually entered venue name (same for the room).